apps/web/app/hooks/features/useTimelogFilterOptions.ts

```
@@ -27,6 +27,14 @@ export function useTimelogFilterOptions() {
         ];
         return user?.role.name ? allowedRoles.includes(user.role.name as RoleNameEnum) : false;
     };
+    const normalizeText = (text: string | undefined | null): string => {
+        if (!text) return '';
+        return text
+            .toLowerCase()
+            .normalize('NFD')
+            .replace(/[\u0300-\u036f]/g, '')
```

Biome (1.9.4)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)